docs/developing/dev_environment/op2_local_proxy_local_solana.md

```diff
@@ -8,8 +8,8 @@ This option will let you connect to Solana via a proxy, when **both** are hosted
 [Docker](https://docs.docker.com/get-docker/) must be already installed on your device. `docker-compose` v1.29 is recommended.
 
 ## Network Configuration
-  * Both the Solana node and the proxy are hosted locally.
-  * The proxy interacts with the Solana node through the Neon EVM.
+  * Both the Solana node and the proxy are hosted locally
```

Comment:
@YYYurchenko This has been updated as a [Gdoc here](https://github.com/neonlabsorg/articles/blob/main/proxy-local/proxy-local.md)

Once we finalize it there for (Medium?) -- can we bring it OR as it is essentially a Get Started, should we rather put it in the proxy local's repo and link from the site to there and retire it from here? WDYT?

BTW I assume that we could improve the docker image to set it to launch all the required services rather than have to do those individually (as per current flow)?

I also assume that we should provide the shell script as a file within the public repo, rather than throw it at users as code in the guide? If so, may I do that (at root??) and update the repo?